src/nimblepkg/options.nim

```diff
@@ -194,6 +194,16 @@ proc prompt*(options: Options, question: string): bool =
   ## forcePrompts has a value different than dontForcePrompt.
   return prompt(options.forcePrompts, question)
 
+proc promptCustom*(options: Options, question, default: string): string =
+  ## Asks an interactive question and returns the result.
+  ##
+  ## The proc will return "default" without asking the user if the global
+  ## forcePrompts is forcePromptYes.
+  result = if options.forcePrompts == forcePromptYes:
```

Comment:
Even though the default is returned, you should still show the prompt and emphasise that the default is chosen. The other prompts do this.

Reply:
Okay. I'll fix this. I think I can add a `ForcePrompt` value to `cli.promptCustom` with a default of `dontForcePrompt`. That way `promptCustom` has all the functionality and the change doesn't break other uses of `promptCustom`. What do you think?

Reply:
Hrm, not entirely certain what you mean, but sounds like it could work. I'll judge it better when I see it, hopefully it's not too much work.

Reply:
I probably didn't explain myself well. Sorry. I'm considering making `cli.promptCusom` similar to `cli.prompt` in that it will accept a `ForcePrompt` argument but have a default of `dontForcePrompt` so other existing uses aren't disrupted.

Reply:
I made the changes. If you don't like them I can can show the prompts another way.